src/Sound.js

```diff
@@ -112,24 +112,28 @@ export default function Sound(props) {
         </button>
       )}
       {sound.previews && (
-        <Waveform
-          waveColor="#FBDC57"
-          backgroundColor="black" barWidth={1}
-          cursorColor="white"  cursorWidth={1}
-          onFinish={() => setIsPlaying(false)}
-          playing={isPlaying}
-          src={sound.previews["preview-lq-mp3"]}
-        />
+        <section>
```

Comment:
Why add a section tag here? If its because of react single root requirement, use react fragments:

https://reactjs.org/docs/fragments.html
https://reactjs.org/docs/fragments.html#short-syntax

Reply:
Ah, I'm familiar! That sounds good. I just thought of `<section>` because that was the most generic HTML tag I could think of; I like the idea of using React fragments, as well.